src/images/TeamPage/arrow.svg

```diff
@@ -0,0 +1,24 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 209 209">
+  <g filter="url(#teampagearrowfilter1)">
+    <circle cx="104.5" cy="104.499" r="73.2871" transform="rotate(-90 104.5 104.499)" fill="url(#paint0_linear)"/>
+  </g>
+  <rect x="64.6538" y="108.384" width="7.51732" height="63.579" rx="3.75866" transform="rotate(-135 64.6538 108.384)" fill="#fff"/>
+  <rect width="7.51731" height="63.5789" rx="3.75866" transform="scale(1 -1) rotate(45 202.997499 120.033306)" fill="#fff"/>
+  <rect x="101.003" y="150.885" width="85.0001" height="7.46599" rx="3.73299" transform="rotate(-90 101.003 150.885)" fill="#fff"/>
+  <defs>
+    <linearGradient id="paint0_linear" x1="31.2128" y1="104.499" x2="177.787" y2="104.499" gradientUnits="userSpaceOnUse">
+      <stop stop-color="#F08C6C"/>
+      <stop offset="1" stop-color="#F06C84"/>
+    </linearGradient>
+    <filter id="teampagearrowfilter1" x=".21283" y=".211487" width="208.574" height="208.574" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+      <feFlood flood-opacity="0" result="BackgroundImageFix"/>
+      <feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
+      <feMorphology radius="9" operator="dilate" in="SourceAlpha" result="effect1_dropShadow"/>
+      <feOffset/>
+      <feGaussianBlur stdDeviation="11"/>
+      <feColorMatrix values="0 0 0 0 1 0 0 0 0 0.7625 0 0 0 0 0.8195 0 0 0 0.56 0"/>
+      <feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/>
+      <feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
+    </filter>
+  </defs>
+</svg>
```

Comment:
We actually don't need to import this twice! We can take the `Arrow` component used for the `EmailSubscription` and rotate it - I've done this on my most recent `Footer` revision, but it might help keep our codebase clean.

Reply:
We might need this one -- not entirely sure if we can get the shadow effect. I will try and get the shadow effect working on the one we have in the codebase now. We'd have to extract the `arrow` component if we are going to reuse it. AKA have it be in a common directory (not buried in `images/Home/...`

Reply:
Well, the `Arrow` component is just an arrow... it's the container/button around it that creates the shadow effect. I'm suggesting that we export that `Arrow` alone and isolate its container styling within a specific component.

You are right though that we'd have to extract its location and move it to a more logically "sharable" place.